Scripts/Mend SCA/generate-async-report.sh

```diff
@@ -101,13 +101,13 @@ if [[ "$MEND_REPORTSCOPE" = "Organization" ]] ; then
 	tokenScope=org
 	token=$MEND_ORG_UUID
 elif [[ "$MEND_REPORTSCOPE" = "Product" ]] ; then
-	[[ -z $WS_PRODUCTTOKEN ]] && echo "No WS_PRODUCTTOKEN specified and MEND_REPORTSCOPE=Product" && exit
+	[[ -z $MEND_PRODUCTTOKEN ]] && echo "No MEND_PRODUCTTOKEN specified and MEND_REPORTSCOPE=Product" && exit
 	tokenScope=product
-	token=$WS_PRODUCTTOKEN
+	token=$MEND_PRODUCTTOKEN
 elif [[ "$MEND_REPORTSCOPE" = "Project" ]] ; then
-	[[ -z $WS_PROJECTTOKEN ]] && echo "No WS_PROJECTTOKEN specified and MEND_REPORTSCOPE=Project" && exit
+	[[ -z $MEND_PROJECTTOKEN ]] && echo "No MEND_PROJECTTOKEN specified and MEND_REPORTSCOPE=Project" && exit
 	tokenScope=project
-	token=$WS_PROJECTTOKEN
+	token=$MEND_PROJECTTOKEN
```

Comment:
This should be changed to $MEND_PROJECT_UUID as it shows in line 23 of the file:
This should be changed because of: https://github.com/mend-toolkit/mend-examples/blob/135433c0e8707ef21fb080853cad0d85578d3aab/Scripts/Mend%20SCA/generate-async-report.sh#L23
To use MEND_PRODUCT_UUID as it shows in line 23 of the file.

Reply:
Updated to use TOKEN instead of UUID.